app/models/list.rb

```diff
@@ -11,6 +11,9 @@ def bought
     def unbought
       where(bought: false)
     end
+    def unbought_and_active
+      where(bought: false).where(deleted_at: nil)
```

Comment:
You can call the other scopes from this scope.  The benefit of doing this is that you only have one source of truth for what `unbought` and `active` mean.
```suggestion
      unbought.active
```